eoscostarica/proton-affiliate
# Implement exchange rate

### Implement exchange rate 

### What does this PR do?

- Resolve #44

### Steps to test

1. make run
2. follow the referral process until validate the KYC
3. confirm the payment with an admin user
4. validate that the amount send match the rate set in params table

#### CheckList
